bootstrapvz/providers/ec2/tasks/ami.py

```diff
@@ -125,3 +125,12 @@ def run(cls, info):
             registration_params['sriov_net_support'] = 'simple'
 
         info._ec2['image'] = info._ec2['connection'].register_image(**registration_params)
+
+        # Setting up tags on the AMI
+        if 'tags' in info.manifest.data and len(info.manifest.data['tags']) > 0:
+            def apply_format(v):
```

Comment:
TBH I'm not sure what do you mean by the above. I can get rid of lambda but not sure if it will improve readability